### PR TITLE
SPCTR-3497: Fix - Content Timeline static styles getting applied to other nested blocks

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

If you add a timeline block additional styles are queued and they are not only affecting the timeline block but all other blocks as well.
https://brainstormforce.atlassian.net/browse/SPCTR-3497

### Screenshots

https://user-images.githubusercontent.com/65082532/279468637-0e9d3a66-16e5-4959-8fc9-4bfb1022b00c.png

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,6 +222,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 * Fix: Icon List - Resolved an issue wherein the vertical layout alignment would not appear as intended in the editor's tablet view.
 * Fix: FAQ - Resolved an issue wherein the block content padding would appear differently in the editor and the front-end.
 * Fix: Global - Resolved an issue wherein the WooCommerce and custom template-styles would not load as intended in FSE theme.
+* Fix: Content Timeline - Resolved an issue wherein the static styles of Content Timeline getting applied to other nested blocks.
```
Comment (bsf-zanev) [inline README.md:225]:
```suggestion
* Fix: Content Timeline - Resolved an issue wherein the block's styling would be applied to other nested blocks in the editor.
```

File: src/blocks/timeline/content-timeline/editor.lazy.scss [hunk 2]
```diff
@@ -2,11 +2,6 @@
   Content Timeline.
 */
 .block-editor-inner-blocks {
```
Comment (AbhijitNage123) [inline src/blocks/timeline/content-timeline/editor.lazy.scss:4]: @rajmohanreddy774 After removing this CSS Please confirm everything works fine in block-editor/ widget editor and in customizer header/footer/widgets.

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: Done.

File: src/blocks/timeline/content-timeline/editor.lazy.scss (hunk 2 above)
Comment (rajmohanreddy774) [reply src/blocks/timeline/content-timeline/editor.lazy.scss:4]: Tested and works fine as expected.

**Review by AbhijitNage123 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**